indexer/src/transform.test.ts

```diff
@@ -0,0 +1,1105 @@
+import {
```

Comment:
I feel like there is a lot of boiler plate code in the tests, I think you can extract 1 or 2 helper functions in order to reduce the amount of code in the file

Reply:
fixed, thanks